packages/core/getUserAvailability.ts

```diff
@@ -126,7 +126,7 @@ export async function getUserAvailability(
     throw new HttpError({ statusCode: 400, message: "Invalid time range given." });
   }
 
-  const where: Prisma.UserWhereUniqueInput = {};
+  const where: Prisma.UserWhereInput = {};
   if (username) where.username = username;
```

Comment:
if username we will need to take into account org also

Reply:
Done, thanks.